src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -0,0 +1,14 @@
+import useToggle from '#hooks/useToggle';
+
+const CanonicalAd = () => {
+  const { enabled: adsEnabled } = useToggle('ads');
+
+  if (!adsEnabled) {
+    return null;
+  }
+
+  // To be implemented in next feature ticket
```

Comment:
Might be worth adding a link to the relevant issue here.

Reply:
Done :)